src/core/manager/workspaces.rs

```diff
@@ -168,24 +168,50 @@ impl Workspaces {
         &mut self,
         wix: usize,
         region: Region,
+        fullscreen_region: Region,
         clients: &[&Client],
-    ) -> Result<(LayoutConf, ArrangeActions)> {
+    ) -> Result<(LayoutConf, bool, ArrangeActions)> {
```

Comment:
I would prefer if `borderless` were a member of `LayoutConf`, like how `gapless` is now, but that struct is a public API and I think that would be a breaking change.